Project2/style.css

```diff
@@ -0,0 +1,25 @@
+.card {
+    max-width: 500px;
+    margin: 10px auto;
+    background-color: #ffffff;}
+figure {margin: 0 auto;}
+        max-width: 500px;
+.button1 {
+    background-color: #008CBA;
+    border: none;
```

Comment:
If you delete the `button` element and move this class to the surrounding `a` element, you can delete this declaration:
```suggestion
```